src/V3Clock.cpp

```diff
@@ -151,12 +175,58 @@ class ClockVisitor final : public VNVisitor {
         clearLastSen();
     }
 
+    // Create sampled values
+    virtual void visit(AstNodeModule* nodep) override {
+        VL_RESTORER(m_modp);
+        {
+            m_modp = nodep;
+            iterateChildren(nodep);
+        }
+    }
+    virtual void visit(AstTopScope* nodep) override {
```

Comment:
This function is unnecessary. The scope underneath will be visited anyway.

Reply:
Thanks, I wasn't sure about that one. Deleted.